lib/index.js

```diff
@@ -69,6 +69,13 @@ internals.Connection.prototype.start = function (callback) {
 
 internals.Connection.prototype.stop = function () {
 
+    // Clean up pending eviction timers.
+    for (var segment in this.cache) {
```

Comment:
Sure, no problem. Should this be updated in the [Hapi style guide](https://github.com/hapijs/contrib/blob/master/Style.md#for-loops)?